src/Documentation/doc/advanced/corporation/quality.md

```diff
@@ -0,0 +1,60 @@
+# Quality
+
+## Basic term
+
+Let's define some terms:
+
+- `AvgInputQuality`: average quality of input materials.
+- `MaxOutputQuality`: maximum value of output material's quality.
+- `OutputQuality`: final value of output material's quality. This value is always less than or equal to `MaxOutputQuality`.
+- `MaxOutputRating`: maximum value of output product's rating.
+- `OutputRating`: final value of output product's rating. This value is always less than or equal to `MaxOutputRating`.
+
+Each industry has a set of input materials and their coefficients, for example, Agriculture needs Water and Chemicals, their coefficients are [0.5, 0.2] respectively. These coefficients do not affect `AvgInputQuality`. `AvgInputQuality` is the mean value of qualities of input materials in warehouse. For example, if Agriculture division's warehouse has Water (quality 1) and Chemicals (quality 11), `AvgInputQuality` is (1+11)/2.
+
+Purchased material is low-quality. Its quality is always 1.
+
+When you import/export your materials between different divisions, you can see quality of some input materials constantly change. Quality is high after EXPORT state, but it reduces after PURCHASE state. Qualities of the materials in warehouse are recalculated in these 2 states.
+
+- Formulas:
+  - In PURCHASE state: $Quality = \frac{Quality\ast CurrentQuantity + BuyAmount}{CurrentQuantity + BuyAmount}$
+    - Our material's quality is "diluted" by low-quality purchased material (quality 1).
+  - In EXPORT state: $Quality = \frac{Quality\ast CurrentQuantity + ImportQuality\ast ImportAmount}{CurrentQuantity + ImportAmount}$
+- This means that the production capability of support division should be balanced. The `ImportAmount` (the number of material units that support division exports) does not need to equal the required number of input material units, but it should also not be too small.
+
+PRODUCTION state uses the "diluted" quality value to calculate `AvgInputQuality` because that state is right after PURCHASE state.
+
+## Material
+
+`MaxOutputQuality` is sum of 3 values:
+
+- Engineer summand:
+  - $EngineerProduction = office.employeeProductionByJob["Engineer"]$
```

Comment:
The quotes are *not* working out well here when rendered, in general this whole expression feels like it wants to be a code block instead of a math block.